recipes/strawberry-graphql/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "strawberry-graphql" %}
+{% set version = "0.114.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/strawberry-graphql-{{ version }}.tar.gz
+  sha256: 242fa8c4354347fa7e92e92f2e2b9a45b6634743cdfce57b2a8b0a608e96021a
+
+build:
+  entry_points:
+    - strawberry = strawberry.cli:run
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
```

```suggestion
    - python >=3.7
```
The upper bound is set by conda-forge once we get there.

Ref: https://github.com/strawberry-graphql/strawberry/blob/c88647514cf78c17b151ee6e8e0a17afc89c6ded/pyproject.toml#L25